Web/Element/HeroImage.hs

```diff
@@ -13,6 +13,7 @@ render record@RenderHeroImage {title, subtitle, button, imageUrl} =
         ++ subTitleWrapped
         ++ buttonHtml
         |> wrapVerticalSpacingBig AlignNone
+        -- 1536x466 - size of the image.
```

Comment:
Sorry, I meant to ask where those values come from. e.g. Why 1536px and not 1537px :)

Reply:
It's from Drupal starter

Reply:
Ok.

```suggestion
```